package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-dialog",
-  "version": "6.2.0",
+  "version": "6.3.0",
```

Comment:
No need to manually bump the version number: it will be bumped automatically from the CI once merged and released